admin/src/server.ts

```diff
@@ -1,69 +1,70 @@
-/* @flow */
-
 import fs from 'fs'
-import logger from 'winston'
 import childProcess from 'child_process'
 import Ajv from 'ajv'
+import { Config, logger } from './config'
+
+function runSubprocess(
```

Comment:
This should return one of the following:

```
{ statusCode: number, status: { result: string } }
```
or
```
{ statusCode: number, error: string }
```

Not sure if there's a way to capture this in Typescript?  Also, I'm happy to change the return type to something simpler if it would get us better type annotations.